components/converter/AccountModule.js

```diff
@@ -1,19 +1,19 @@
 import React, { useRef } from 'react'
 import PropTypes from 'prop-types'
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
 import { OverlayTrigger, Popover } from 'react-bootstrap'
-import { useWeb3Connect } from '../../web3-connect'
+import { useWeb3Connect } from 'lib/web3-connect'
 import {
   useJurorRegistryAnjBalance,
   useEthBalance,
```

Comment:
Let's alphabetize this!

Reply:
Done in https://github.com/aragon/anj.aragon.org/pull/23/commits/1002e09f752f58c3d7bf1898f54a89c8a403bfa1